cliquet/views/heartbeat.py

```diff
@@ -9,9 +12,27 @@
 def get_heartbeat(request):
     """Return information about server health."""
     database = request.db.ping()
+    cache = request.cache.ping()
 
-    status = dict(database=database)
+    settings = request.registry.settings
+    server_url = settings['fxa-oauth.oauth_uri']
+
+    oauth = None
+    if server_url is not None:
+        auth_client = OAuthClient(server_url=server_url)
+        server_url = auth_client.server_url
+        oauth = False
+
+        try:
+            r = requests.get(urljoin(server_url, '/__heartbeat__'), timeout=10)
+            r.raise_for_status()
+            oauth = True
+        except requests.exceptions.HTTPError:
+            pass
```

Comment:
In order to keep this heartbeat agnostic, I would try to move this in the authentication module.

I'm not sure we can do something like this:

``authent = registry.authent_policy.ping()``

Where ``ping()`` would be a method on ``Oauth2AuthenticationPolicy`` class.


Otherwise, maybe just define a ``fxa_ping()`` method in ``cliquet.authentication`` to keep things all together there.

What do you think ?